index.js

```diff
@@ -25,3 +29,17 @@ module.exports = function (fn, P) {
 		});
 	};
 };
+
+pify.all = function (module, P) {
+	var pifyModule = {};
+
+	for (var method in module) {
+		if (isFn(module[method])) {
```

Comment:
Maybe also make sure it's not an inherited function?

`if (isFn(module[method]) && ({}).hasOwnProperty.call(module, method)) {`